config_examples/config_evaluate.yaml

```diff
@@ -0,0 +1,29 @@
+project: mitotic_classifier
+
+model_params:
+  name: 'resnet'
+  depth: 50
+  in_channels: 2
+  num_classes: 9
+  class_weight: [1, 1, 1, 1, 1, 1, 1, 1, 1]
+  class_label: ["M0", "M12", "M3", "M45", "M67_e", "M6M7_h", "blob", "dead", "wrong"]
+  trained_model: 
+    - model: "/allen/aics/assay-dev/users/Jianxu/Projects/mitotic_classifier_training_data_retrospect/logs/lightning_logs/version_5/checkpoints/epoch=7-step=895.ckpt"
```

Comment:
Is it alright for this repo to have full path like: `/allen/aics/assay-dev/users/Jianxu` hard coded?

Reply:
good catch!!! I try very hard to avoid this. But there are still some remaining .... Will fix this!